.github/workflows/build-test.yml

```diff
@@ -120,17 +120,30 @@ jobs:
       with:
         ipfs_version: ^0.8
 
-    - name: Pin testing data
+    - name: Prepare testing data to pin on IPFS
+      run: |
+        mkdir itk-wasm-testing-data
+        rsync -v -r --exclude=Objects --exclude='*-hash-stamp' ./build/ExternalData/ ./itk-wasm-testing-data/
```

Comment:
Todo:

- Checkout DataLad Repository
- Rsync updates
- Check whether anything has changed.
-- If so, push to gin/github
-- Set a variable to determine whether to pin to web3.storage / pinata